lib/common.sh

```diff
@@ -274,12 +274,7 @@ determineTool() {
     if [ -f "${goMOD}" ]; then
         TOOL="gomodules"
         warn ""
-        warn "Go modules are an experimental feature of go1.11+"
-        warn "Any issues building code that uses Go modules should be"
-        warn "reported via: https://github.com/heroku/heroku-buildpack-go/issues"
-        warn ""
-        warn "Additional documentation for using Go modules with this buildpack"
-        warn "can be found here: https://github.com/heroku/heroku-buildpack-go#go-module-specifics"
+        warn "Found a go.mod file and using Go modules"
```

Comment:
I don't think we want the `!!!` in the message here. I would suggest something like:

```
step "Detected go modules - go.mod"
```

and when we drop support for other package managers we can delete this line

Reply:
Excellent. Yes will do this.